### PR TITLE
Tell Spring REST that these services only produce application/json

### PR DESCRIPTION
**Pull request by eltmon**
This is to address https://github.com/ExpediaDotCom/seiso/issues/196

### CODE REVIEW

File: src/main/java/com/expedia/seiso/web/controller/EnvironmentController.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import com.expedia.seiso.web.resource.ServiceInstanceResource;
 
 @BasePathAwareController
-@RequestMapping("/environments")
+@RequestMapping(value = "/environments", produces="application/json")
 public class EnvironmentController {
```
Comment (launchcg-ztonia) [inline src/main/java/com/expedia/seiso/web/controller/EnvironmentController.java:16]: Environment Controller did not have this issue

File: src/main/java/com/expedia/seiso/web/controller/ServiceController.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
 import com.expedia.seiso.web.resource.ServiceInstanceResource;
 
 @BasePathAwareController
-@RequestMapping("/services")
+@RequestMapping(value = "/services", produces="application/json")
```
Comment (launchcg-ztonia) [inline src/main/java/com/expedia/seiso/web/controller/ServiceController.java]: ServiceController did not have this issue

File: src/main/java/com/expedia/seiso/web/controller/ServiceInstanceController.java [hunk 3]
```diff
@@ -49,7 +49,7 @@
 //@BasePathAwareController
 @RestController
 //@RequestMapping(value = "/serviceInstances")
-@RequestMapping(value = "/api/serviceInstances")
+@RequestMapping(value = "/api/serviceInstances", produces="application/json")
```
Comment (launchcg-ztonia) [inline src/main/java/com/expedia/seiso/web/controller/ServiceInstanceController.java]: Works great.
Might want a more generalized solution.

**Comment by eltmon** [comment]
Removed from other two controllers per Zaal's feedback.

**Pull request merged**